### ISSUE TITLE
Better error when C(++) compiler cannot be found

### ISSUE THREAD

**Issue by dom96** [nim-lang/Nim#2135]
The compiler just crashes whenever the C compiler is not found:

```
Traceback (most recent call last)
nim.nim(92)              nim
nim.nim(56)              handleCmdLine
main.nim(255)            mainCommand
main.nim(65)             commandCompileToC
extccomp.nim(636)        callCCompiler
osproc.nim(263)          execProcesses
osproc.nim(158)          startCmd
osproc.nim(470)          startProcess
os.nim(277)              raiseOSError
Error: unhandled exception: The system cannot find the file specified.
 [OSError]
```

**Comment by DTSCode** [comment]
oh yay! something even I can fix

**Comment by Yardanico** [comment]
@Araq seems to be fixed by #5663
